src/preload/main-window.js

```diff
@@ -71,24 +71,37 @@ const runtimeApi = {
 	setLocale: async (value) => {
 		return ipcRenderer.invoke('settings:set:locale', value)
 	},
+
+	// Sentry
+	getSentryUserId: () => {
+		return getProcessArgValue('comapeo-sentry-user-id')
+	},
+	enableSentryOnInit: () => {
+		return getProcessArgValue('comapeo-sentry-enable') === 'true'
+	},
```

Comment:
suggestions: combine into `getSentryConfig()`, because we are going to be adding more things with the app usage metrics and that will be easier to maintain and keep both processes in sync.